src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div class="wrap">
+    <h3> Arm controls </h3>
+    <div class="controls">
+      <Checkbox ref="arm-enabled" v-bind:name="'Arm Enabled'" v-on:toggle="updateArmEnabled($event)"/>
+    </div>
+  </div>
+</template>
+
+<script>
+import ROSLIB from 'roslib'
+import Checkbox from './Checkbox.vue'
+import { mapGetters, mapMutations } from 'vuex'
+
+
+let interval;
```

Comment:
nit: Seems silly to keep these interval variables around if we're not going to use them. I think we can get rid of this and just call `window.set_interval()` below

Reply:
We use it here to ensure the interval is destroyed with the object 
`beforeDestroy: function () {
    window.clearInterval(interval)
  },`